client/my-sites/checkout/composite-checkout/hooks/use-create-payment-methods/index.ts

```diff
@@ -62,7 +66,15 @@ export function useCreateCreditCard( {
 	shouldUseEbanx,
 	shouldShowTaxFields = false,
 	activePayButtonText = undefined,
-} ) {
+}: {
+	isStripeLoading: boolean;
+	stripeLoadingError: StripeLoadingError;
+	stripeConfiguration: StripeConfiguration | null;
+	stripe: Stripe | null;
+	shouldUseEbanx: boolean;
+	shouldShowTaxFields?: boolean;
+	activePayButtonText?: string | undefined;
```

Comment:
Some uses of `activePayButtonText` in non-ts files aren't cast to string.

Reply:
Yes, you're right. This is an issue but I'm not super concerned about it here. We'll need to handle those when we convert those files to TS and there will probably be lots of similar issues. I'm not worried because it shouldn't cause any issues as long as the `translate()` is just producing a string (most uses of `translate()` do) and not a React component. This will be a lot easier when we start switching to `react-i18n` which separates the two concepts.